specification/common-types/resource-management/v2/types.json

```diff
@@ -659,7 +669,7 @@
       "in": "path",
       "required": true,
       "type": "string",
-      "description": "The name of the resource group. The name is case insensitive.",
+      "description": "The name of the resource group.",
```

Comment:
any reason to remove the sentence?

Reply:
Hi there was a conflict error where the description is different between this and the other version. I am fairly new to this but I believe I either have to fix the description here, or wherever they appear in other files. Given this is only a description, and that changing it here or at other places (I went ahead and fix it here as it is just one place).

Reply:
@zhenglaizhang could you please advise us what is the next step here? We have addressed all issues related to our changes. Thank you!

Reply:
do you have the link to the conflict error? I need to check what's the detail.

Reply:
@zhenlaizhang, here is the error with the description.
"readme":"monitor/resource-manager/readme.md",
"tag":"package-2021-09",
"details":"Error: '$.parameters.ResourceGroupNameParameter.description' has incompatible values (---The name of the resource group.,
---The name of the resource group. The name is case insensitive.)."

Reply:
> do you have the link to the conflict error? I need to check what's the detail.

Could you please advise the next step here? This PR has been in pending for too long. Thanks!